### DISCUSSION TITLE
Welcome to nextjs-showcase Discussions!

### DISCUSSION THREAD

**Discussion by emmanuelgautier** [emmanuelgautier/nextjs-showcase#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nvima** [comment]
Hey, i used your repository to build my ssg app.
But got one Problem. 
https://delivery.plus/about
https://de.delivery.plus/about
https://fr.delivery.plus/about
https://es.delivery.plus/about

if i refresh the page or switch to the next page via an a href, the correct language remains. but when i change a page via next/link, the default language is always selected, which is german. Do you have any tips on how I can fix this? I'm not very familiar with next so far.

I also set the respective subdomains correctly as in: https://nextjs.org/docs/advanced-features/i18n-routing#getting-started

**Comment by nvima** [comment]
I was able to fix the problem.
I exchanged the env variable with NEXT_PUBLIC_LOCALE. I exchanged these in _document and _app. so it's now running local and live.

I would still like to adapt the routes to the language, but I don't think it's that easy, what do you think about it?
for example:
https://delivery.plus/contact
https://de.delivery.plus/contact  -> kontakt
https://fr.delivery.plus/contact  -> contact
https://es.delivery.plus/contact  -> contacto
